### DISCUSSION TITLE
Do I need an external WLAN card?

### DISCUSSION THREAD

**Discussion by dgawin** [WLAN-Pi/wlanpi-profiler#72] [Q&A] [answered]
Hello, do I need an external WLAN card for comparable captures? Or is the WLAN chip from the WLANPI sufficient for this?

Regards,
Dominik

**Comment by joshschmelzle** [comment, marked as answer]
The requirement is a WLAN interface (and driver) which supports both monitor mode and packet injection.

That includes mt76x2u (recommended) and rtl88XXau adapters are tested regularly (everything else is experimental and not officially supported).

EDIT: AX200/AX210 are also supported now.

EDIT 2: MT7921K/MT7921AU are also supported now.
